mapmssql2008.c

```diff
@@ -1446,32 +1450,77 @@ static int prepare_database(layerObj *layer, rectObj rect, char **query_string)
   /* start creating the query */
 
   query = msStringConcatenate(query, "SELECT ");
+  if (layerinfo->paging && (layer->maxfeatures >= 0 || layer->startindex > 0))
+    paging_query = msStringConcatenate(paging_query, "SELECT ");
 
   /* adding items to the select list */
   for (t = 0; t < layer->numitems; t++) {
 #ifdef USE_ICONV
       query = msStringConcatenate(query, "convert(nvarchar(max), [");
       query = msStringConcatenate(query, layer->items[t]);
-      query = msStringConcatenate(query, "]),");
+      query = msStringConcatenate(query, "]) '");
```

Comment:
I'm probably missing something but it looks the #ifdef USE_ICONV and the #else branch code are strictly identical, except the initial line. So this could be largely simplified by placeing just it in the #if #else #endif, and have the rest to be common conde

Reply:
yes, that could be simplified in that way. 
However I don't understand the reason of the code change above.

Reply:
Fixed